api/lib/classes/sendGrid.js

```diff
@@ -0,0 +1,20 @@
+class SendGrid {
+  #apiKey // this is a private property that will not be able to be accessed outside of this class definition
```

Comment:
```suggestion
  #sendGrid
```
Why sendgrid is private:
```
const sendGrid = new SendGrid( apikey );

sendGrid.sendGrid; //syntax error (can only be in out brackets);